caom2utils/caom2utils/fits2caom2.py

```diff
@@ -2574,28 +2588,32 @@ def _get_algorithm(self, obs):
         # TODO DEFAULT VALUE
         name = self._get_from_list('Observation.algorithm.name', index=0,
                                    current=obs.algorithm.name)
-        self.logger.debug('End Algorithm augmentation.')
+        result = None
         if name:
-            return Algorithm(str(name))
-        else:
-            return None
+            result = Algorithm(str(name))
```

Comment:
could be replaced with a one liner:
`result = Algorithm(str(name)) if name else None`

Reply:
Replaced.